main/Select/Item.tsx

```diff
@@ -0,0 +1,56 @@
+import { TextStyle, ViewStyle } from 'react-native';
+
+import React from 'react';
+import { THEME } from './theme';
+import styled from 'styled-components/native';
+
+interface Props {
+  testID?: string;
+  isDarkMode?: boolean; // dark mode
+  activeOpacity?: number; // set the opacity of selected value
+  containerHeight?: number; // fix the height of dropdown list
+  value: string | number; // current selected value
+  onSelectItem?:((param: string | number) => void) | undefined;
+  style?: ViewStyle | TextStyle;
```

Comment:
I hope I don't want to meet this situation. Please separate the styles.
`containerStyle`, `textStyle`, you should specify styles for different components separately with explicit namings.